tests/applications/ymir_detectors.json

```diff
@@ -1576,8 +1576,7 @@
           ],
           "name": "daq_publication",
           "type": "group"
-        },
-        "$USERS$"
```

Comment:
Can you add a note of this modification in the related docstring...?

Reply:
Do you mean in the docstring of the `ymir` fixture?